_posts/2019-06-14-blt-0.2.5.md

```diff
@@ -10,5 +10,5 @@ LLNL initially released BLT in 2017, and now v0.2.0 and v0.2.5 are available.
 Learn more:
 - [Release notes](https://github.com/LLNL/blt/releases)
 - [GitHub repo](https://github.com/LLNL/blt)
-- [Documentation](https://llnl-blt.readthedocs.io/en/latest/)
+- [Documentation](https://llnl-blt.readthedocs.io/en/develop/)
```

Comment:
Perhaps "master" would be a better substitute for the "latest" version, rather than the development version?
https://llnl-blt.readthedocs.io/en/master/